### PR TITLE
Add fidelity to qubit and fix typo

### PR DESCRIPTION
**Pull request by andrea-pasquale**
This PR adds a `fidelity` argument to qubit and it also fixes the a typo `assigment_fidelity` -> `assignment_fidelity`.
I spotted this errors while working on qiboteam/qibocal#548.
I will open also a PR in qibolab_platform_qrc.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

File: src/qibolab/qubits.py [hunk 1]
```diff
@@ -57,7 +57,8 @@ class Qubit:
     g: float = 0.0
     """Readout coupling"""
 
-    assigment_fidelity: float = 0.0
+    assignment_fidelity: float = 0.0
+    fidelity: float = 0.0
```
Comment (stavros11) [inline src/qibolab/qubits.py:61]: Maybe it's better to use something like
```suggestion
    measurement_fidelity: float = 0.0
```
to distinguish from potential gate fidelities, etc. that maybe we can calculate in the future?

**Review by stavros11 — APPROVED** [review]
Thanks for the fix!

File: src/qibolab/qubits.py (hunk 1 above)
Comment (andrea-pasquale) [reply src/qibolab/qubits.py:61]: Indeed I also think that `fidelity` is a bit too generic.
Is there a specific term for the fidelity obtained using single shot classification, @Edoardo-Pedicillo @Jacfomg ?
In our case the fidelity is given by this
https://github.com/qiboteam/qibocal/blob/3bd6404e6c9ab35e725d219a6630bb6bd8a9794a/src/qibocal/fitting/classifier/qubit_fit.py#L105
Maybe we can call it `readout_fidelity`?

**Review by Edoardo-Pedicillo — APPROVED** [review]

File: src/qibolab/qubits.py (hunk 1 above)
Comment (Jacfomg) [reply src/qibolab/qubits.py:61]: Yup, it's called that if I am not mistaken.

Comment (stavros11) [reply src/qibolab/qubits.py:61]: Great, then let's go with `readout_fidelity`. @andrea-pasquale feel free to merge after you make the change.

Comment (andrea-pasquale) [reply src/qibolab/qubits.py]: Thanks for the feedback. I will merge now.

**Pull request merged**